plugins/org.eclipse.elk.graph/src/org/eclipse/elk/graph/util/ElkGraphUtil.java

```diff
@@ -439,7 +440,57 @@ public static ElkNode findLowestCommonAncestor(final ElkNode node1, final ElkNod
         return commonAncestor;
     }
 
+    /**
+     * Returns the {@link ElkNode} corresponding to the {@code childPath}.
+     *
+     * @param parent
+     *            a parent in the hierarchy of the wanted child.
+     * @param childPath
+     *            a list of identifier ({@link ElkShape#getIdentifier()}) to go from {@code parent} to the wanted child.
+     * @return the {@link ElkNode} corresponding to the {@code childPath}
+     * @throws NullPointerException
+     *             if {@code parent} is {@code null}.
+     * @throws IllegalArgumentException
+     *             if the {@code childPath} does not match to an existing path
+     */
+    public static ElkNode getChild(final ElkNode parent, final String... childPath) {
```

Comment:
I'm not exactly sure about this method (and the next). I'm not questioning its usefulness, but I feel like it should only be used in a unit test context, and thus might be moved to `GraphTestUtils`. @uruuru, what do you think?

Reply:
OK, I will move these methods to GraphTestUtils. I've imagined that this could be useful even elsewhere that in tests.

Reply:
Done